sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -66,6 +66,9 @@
 - Rename `Models::DeleteSnapshotsOption::Only` to `Models::DeleteSnapshotsOption::OnlySnapshots`.
 - Rename `SourceConditions` in API options to `SourceAccessConditions`.
 - Remove Blob Batch.
+- Add `CreateBlobContainer`, `DeleteBlobContainer`, `UndeleteBlobContainer` into `BlobServiceClient`.
+- Add `DeleteBlob` into `BlobContainerClient`.
```

Comment:
How is Adding APIs a breaking change? if it is, maybe explain it here

Reply:
fixed